### DISCUSSION TITLE
Oligomer Tool

### DISCUSSION THREAD

**Discussion by Mclnol16** [michaelmarty/UniDec#65] [Q&A]
Dear Dr. Michael Marty,

Thank you a for this program and its continued development. Its been super useful in my analysis of my proteins.

I have a questions/request about the Oligomer Tool. Is there a way to show all the possible combinations in the oligomer list and their respective errors? Right now, it only shows the best possible combination and this is often wrong. For instance, my protein can form several multimeric states and each one of these combinations may contain several; salts, disulfide bonds, and ptms. When inputing this into the list, it will often suggest combinations that are not possible (Several PTMS on a single protein combined with a large number of salt). If there was a way to see all the combinations with some error cutoff (or filtering option), then I could make the judgement on whether or not the assignment is real.

Thank you
Regards,
Nolan (WashU grad student)

**Comment by michaelmarty** [comment]
Great suggestion, Nolan! I'm going to be out of town for the next two weeks, but I'll try to throw something together after that. Would you have any data you'd like to have me test it on? Thanks,
MTM

**Reply by Mclnol16** [reply]
Happy to share some data.

Is there a particular way you would want me to send it to you?

Nolan

**Reply by michaelmarty** [reply]
Perfect. Just email it as an attachment or a download link. Whatever is easiest. Thanks!
